server/server.js

```diff
@@ -0,0 +1,71 @@
+//user Sign Up
+Meteor.methods({
+    user_signup: function(data) {
+        Accounts.createUser({
+            email: data.email,
+            password: data.id,
+            profile: {
+                social_id: data.id,
+                social_type: data.social_type,
+                name: data.name,
+                picture: data.pic,
+                friends: data.friends,
+                geo: data.geo,
+                geo_update_time: data.geo_update_time,
+                created_at: Date.now()
+            }
+        });
+
+    }
+});
+
+
+//user validation before Login and Signup
+Meteor.method({
+    validate_user: function(user) {
```

Comment:
this should not be a seperate api function. 
from client/ side this function will not be called. remove it